apps/www/components/icons.tsx

```diff
@@ -1,7 +1,11 @@
 type IconProps = React.HTMLAttributes<SVGElement>
 
-export const Icons = {
-  logo: (props: IconProps) => (
+type IconComponents = {
+  [key: string]: (props: IconProps) => JSX.Element
```

Comment:
`[key: string]` will disable TypeScript intellesence auto-completion.

Try writing `Icons.l`. It won't suggest `logo`. why? because It could be any string.